models/chainlink/ethereum/chainlink_ethereum_ocr_reverted_transactions.sql

```diff
@@ -0,0 +1,58 @@
+{{
+  config(
+    tags=['dunesql'],
+    alias='ocr_reverted_transactions',
+    partition_by = ['block_time'],
```

Comment:
It's not possible to partition by `block_time`. This will create one partition per block_time, and likely an enormous amount of partitions.

You likely want to add an additional column `block_date`, defined as `cast(block_time as date) as block_date`, and partition by `block_date` instead.